Jenkinsfile_CNP

```diff
@@ -2,6 +2,12 @@
 //noinspection GroovyUnusedAssignment
 @Library("Infrastructure") _
 
+def branchesToSync = ['demo', 'perftest']
+
 withInfraPipeline("ds") {
   enableSlackNotifications("#family-integration-cicd-notice")
 }
+
+withPipeline(type, product, component) {
+  syncBranchesWithMaster(branchesToSync)
```

Comment:
put this inside withInfraPipeline and remove withPipeline

Reply:
ok will do this